### PR TITLE
update: hatch tutorial

### PR DESCRIPTION
**Pull request by ucodery**
addressed #212

### CODE REVIEW

File: tutorials/get-to-know-hatch.md [hunk 1]
```diff
@@ -4,15 +4,35 @@ Our Python packaging tutorials use the tool Hatch.
...
+:::
 
 :::{tip}
-If you are comfortable using [pipx](https://pipx.pypa.io/stable/) to install hatch, we encourage you to do so. pipx will ensure that your package is available across all of your Python environments on your computer rather than just in the environment that you install it into.
+Hatch can also be installed directly using `pip` or `conda`, but we encourage you to use `pipx`.
+This is because `pipx` will ensure that your package is available across all of your Python
+environments on your computer rather than just in the environment that you install it into.
 
-However if you are not sure about pipx, you can install hatch using pip or conda.
+If you install hatch this way you will have to take care that the environment it is installed into
```
Comment (willingc) [inline tutorials/get-to-know-hatch.md:23]: Should this be a continuation of the pip/conda paragraph to make it clear that it isn't referring to pipx?

**Review by willingc — COMMENTED** [review]
Nice! Thanks @ucodery.

File: tutorials/get-to-know-hatch.md (hunk 1 above)
Comment (ucodery) [reply tutorials/get-to-know-hatch.md:23]: If that would that make it clearer then yes. I did see some ambiguity here, but it also felt weird to write out `pipx` so many times in one block.

Comment (willingc) [reply tutorials/get-to-know-hatch.md:22]: Thanks!

**Review by lwasser — APPROVED** [review]
i'm such a fan of the :shipit: 🚢 let's merge!

**Pull request merged**